geode-core/src/main/java/org/apache/geode/internal/cache/EntryExpiryTask.java

```diff
@@ -328,8 +328,8 @@ protected void basicPerformTimeout(boolean isPending) throws CacheException {
           if (logger.isTraceEnabled()) {
             // NOTE: original finer message used this.toString() twice
```

Comment:
I don't this comment is necessary anymore.

Reply:
will remove